test/tilt_template_test.rb

```diff
@@ -308,6 +306,7 @@ def default_encoding
     end
 
     test "uses the template from the generated source code" do
+      Encoding.default_external = 'UTF-8'
```

Comment:
Should we set this in `test_helper.rb` instead? Looks a bit odd to modify a global var inside a single test.

Reply:
This practice is already done in all of the other tests in this section.  The teardown block resets it to the previous default.

I thought of setting this only in test_helper, but then we run all tests with UTF-8 default external encoding, which is a bad idea as it will not allow us to easily catch regressions introduced when the default external encoding is not-UTF8.

Reply:
Ugh, I only looked at the diff, not the full file. Sorry!